docs/django-apps.md

```diff
@@ -0,0 +1,87 @@
+# Using Caluma as django apps
+
+## Disclaimer
+
+It is highly recommended to use Caluma as a dedicated service. However, there are usecases, where
+it might make sense to integrate Caluma into another django project.
+
+If you just want to get Caluma up and running, please see the documentation about [setting up
+the Caluma service](configuration.md).
+
+Please beware that Caluma only works with PostgreSQL, and requires the `psqlextra`
+backend to use the advanced features (such as JSON fields etc).
+
+## Installation
+
+Caluma is on PyPI, so you can just
+
+```shell
+pip install caluma
+```
+
+
+## Configuration
+
+### Installed apps
+
+Add the Caluma apps you want to use to your `INSTALLED_APPS`.
+
+Some notes about Caluma-internal dependencies:
+
+* `caluma_workflow` needs `caluma_form` to work correctly (as cases and work items point to documents)
+* `caluma_core` should always be installed when using Caluma (though, technically,
+  it would work without it, as core doesn't contain any models
```

Comment:
I'd leave the note in brackets out because it doesn't really matter to someone who just wants to get started with caluma in their django app (same below)

Reply:
See #954.